packages/locale/res/translations/it.json

```diff
@@ -0,0 +1,467 @@
+    ["Your search did not match any items.", "La ricerca non corrisponde a nessun elemento."],
+    ["This vault does not have any items yet.", "Questo archivio non ha ancora elementi."],
+    ["You don't have any attachments yet.", "Non hai ancora nessun allegato."],
+    ["You don't have any favorites yet.", "Non hai ancora nessun preferito."],
+    ["You don't have any recently used items!", "Non hai elementi usati di recente!"],
+    ["You don't have any items yet.", "Non hai ancora nessun elementi."],
+    ["Type To Search", "Digita per ricercare"],
+    ["{0} items selected", "{0} elementi selezionati"],
+    [
+        "Some items in your selection are from Vaults you don't have write access to and cannot be deleted. Do you want to proceed deleting the other items?",
+        "Alcuni elementi nella tua selezione provengono da Archivi a cui non hai accesso in scrittura e non possono essere eliminati. Vuoi procedere con l'eliminazione degli altri elementi?"
+    ],
+    ["Yes", "Si"],
+    ["No", "No"],
+    ["Are you sure you want to delete these items? This action can not be undone!", "Sei sicuro di voler eliminare questi elementi? Questa azione non può essere annullata!"],
+    ["Delete {0} Items", "Elimina {0} Elementi"],
+    [
+        "Some items in your selection have attachments and cannot be moved. Do you want to proceed moving the other items?",
+        "Alcuni elementi nella selezione hanno allegati e non possono essere spostati. Vuoi procedere spostando gli altri elementi?"
+    ],
+    [
+        "Some items in your selection are from Vaults you don't have write access to and cannot be moved. Do you want to proceed moving the other items?",
+        "Alcuni elementi della tua selezione provengono da Archivi a cui non hai accesso in scrittura e non possono essere spostati. Vuoi procedere spostando gli altri elementi?"
+    ],
+    ["No Name", "Nessun nome"],
+    ["copied", "copiato"],
+    ["No Fields", "Nessun campo"],
+    ["This item has no fields.", "Questo elemento non ha campi."],
+    ["Email Address", "Indirizzo Email"],
+    ["Master Password", "Password Principale"],
+    ["Login", "Login"],
+    ["New to Padloc?", "Nuovo in Padlock?"],
+    ["Sign Up Now", "Registrati ora"],
+    ["Please enter a valid email address!", "Inserisci un indirizzo email valido!"],
+    ["Please enter the confirmation code sent to your email address to proceed!", "Inserisci il codice di conferma inviato al tuo indirizzo email per procedere!"],
+    ["One Last Step!", "Un Ultimo Passo!"],
+    ["Enter Verification Code", "Immettere il codice di verifica"],
+    ["Maximum number of tries exceeded! Please resubmit and try again!", "Numero massimo di tentativi superato! Si prega di riprovare!"],
+    ["Wrong username or password. Please try again!", "Nome utente o password errati. Riprova!"],
+    ["Can't remember your master password?", "Non ricordi la tua password principale?"],
+    ["Recover Account", "Recupera Account"],
```

Comment:
Should this have been `Recupera Conto`?

Reply:
'Conto' is more like Bank Account, in Italy we use Account for things like this

Reply:
Thanks for confirming!